File: Dotnet/Task.cs
using System;
using Newtonsoft.Json;

namespace Dotnet
{
    public class Task
    {        
        [JsonProperty("id")]
        public string Id { get; set; }

        [JsonProperty("title")]
        public string Title { get; set; }

        [JsonProperty("description")]
        public string Description { get; set; }

        [JsonProperty("dueDate")]
        public DateTime DueDate { get; set; }

        [JsonProperty("isComplete")]
        public bool IsComplete { get; set; }

        [JsonProperty("parentId")]
        public string ParentId { get; set; }

        [JsonProperty("type")]
        public string Type { get { return "task"; } }

    }
}

File: Go/task.go
package main

import "time"

type Task struct {
	Id string `json:"id,omitempty"`
	Title string `json:"title"`
	Description string `json:"description"`
	IsComplete bool `json:"isComplete"`	
	ParentId string `json:"parentId"`
	Type string `json:"type"`
	DueDate time.Time `json:"dueDate"`
}
